libclamav/cvd.c

```diff
@@ -732,3 +740,37 @@ int cli_cvdunpack(const char *file, const char *dir)
     close(fd);
     return ret;
 }
+
+cl_error_t cl_cvdunpack(const char *file, const char *dir, bool dont_verify)
+{
+    cl_error_t status = CL_SUCCESS;
+    FILE *fs          = NULL;
+
+    fs = fopen(file, "rb");
+    if (NULL == fs) {
+        char err[128];
```

Comment:
Why does this cli_strerror call use err[128] when the others do not? This is not a static function or anything.

Reply:
Our `cli_strerror` is a wrapper for the poorly standardized `strerror` system function, not to be confused with `cl_strerror`, which is the clamav API for converting the clam error enum to a message. 

The `err[128]` approach is something I copypasted from elsewhere in the code.  Such as: https://github.com/Cisco-Talos/clamav/blob/main/libclamav/untar.c#L318-L319